docker-compose.yaml

```diff
@@ -66,3 +66,4 @@ services:
     command: /bin/sh -c "python -u manage.py runserver 0.0.0.0:8000"
     environment:
       DATABASE_URL: postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${POSTGRES_HOST}:${POSTGRES_PORT}/data_store_api
+      ES_HOSTNAME: ${ES_HOSTNAME}
```

Comment:
You are the third developer attempting to add this env var to the docker-compose file. Let's wait for Will/Joe to add an ES container to the yaml file first.

Reply:
It would be helpful to keep this in. If it were _just_ a step in Dockerizing local elasticsearch, then yes, holding off makes sense. However, this is needed to make the standard setup (i.e. running API in docker) work with elasticsearch features -- _wherever_ that elasticsearch may be. 

i.e. you _could_ set this to a dockerized elasticsearch container, but you could also set it to the `sandbox` or `nonprod` cluster endpoints, or your own standalone non-dockerized ES cluster.

Not having this is like not setting the `DATABASE_URL` env var (albeit a much smaller impact).

Reply:
`ES_HOSTNAME` can easily be set in settings.py, unlike `DATABASE_URL`